shared/actions/platform-specific/push.native.tsx

```diff
@@ -347,43 +350,45 @@ function* _checkPermissions(action: ConfigGen.MobileAppStatePayload | null) {
   }
 }
 
-const getStartupDetailsFromInitialPush = (): Promise<
-  | null
-  | {
-      startupFollowUser: string
+function* getStartupDetailsFromInitialPush() {
```

Comment:
Does this need to be a generator? You could do the same thing with `setTimeout` calling `reject` on a promise...